controller/volume_controller.go

```diff
@@ -549,7 +553,7 @@ func (vc *VolumeController) ReconcileEngineReplicaState(v *longhorn.Volume, e *l
 		//   2. the volume is expanding size.
 		if !((v.Status.IsStandby || v.Status.RestoreRequired) && cliAPIVersion < engineapi.CLIVersionFour) &&
 			v.Spec.Size == e.Status.CurrentSize {
-			if err = vc.replenishReplicas(v, e, rs); err != nil {
+			if err = vc.replenishReplicas(v, e, rs, ""); err != nil {
```

Comment:
Why we're using `err = ` instead of `err :=` here?

Reply:
Thank you! Fixed